### PR TITLE
Generalize Apply, removing the need for a separate Apply2

### PR DESCRIPTION
**Pull request by eggrobin**

### CODE REVIEW

File: quantities/tuples.hpp [hunk 1]
```diff
@@ -52,13 +42,9 @@ struct DerivativesGenerator;
...
 using internal_tuples::is_tuple_v;
 
-template<template<typename> class Transform, typename Tuple>
-using Apply = typename internal_tuples::ApplyGenerator<Transform, Tuple>::Type;
-
-template<template<typename, typename> class Transform,
-         typename LTuple, typename RTuple>
-using Apply2 =
-    typename internal_tuples::Apply2Generator<Transform, LTuple, RTuple>::Type;
+template<template<typename> class Transform, typename... Tuples>
```
Comment (pleroy) [inline quantities/tuples.hpp]: Why `class` here and `typename` above?  Also why no `...` after the innermost `typename`?

File: quantities/tuples_body.hpp [hunk 2]
```diff
@@ -33,21 +33,24 @@ struct TupleElementOrVoid<Tuple, index, true> {
...
 
-template<template<typename> class Transform,
-         typename Tuple, std::size_t... indices>
-struct ApplyGenerator<Transform, Tuple, std::index_sequence<indices...>>
-    : not_constructible {
-  using Type = std::tuple<Transform<std::tuple_element_t<indices, Tuple>>...>;
-};
+template<template<typename...> typename Transform,
+         typename... Tuples>
+struct ApplyGenerator {
```
Comment (pleroy) [inline quantities/tuples_body.hpp]: Please make this a class and put everything except `Type` in a private part.

File: quantities/tuples.hpp (hunk 1 above)
Comment (eggrobin) [reply quantities/tuples.hpp]: Done.

File: integrators/integrators.vcxproj [hunk 3]
```diff
@@ -13,6 +13,7 @@
     <ClInclude Include="backward_difference_body.hpp" />
     <ClInclude Include="cohen_hubbard_oesterwinter.hpp" />
     <ClInclude Include="cohen_hubbard_oesterwinter_body.hpp" />
+    <ClInclude Include="embedded_explicit_runge_kutta_integrator.hpp" />
```
[the remark this answers is not in the document]
Comment (eggrobin) [reply integrators/integrators.vcxproj]: Done.

File: quantities/tuples_body.hpp (hunk 2 above)
Comment (eggrobin) [reply quantities/tuples_body.hpp]: Done.

**Pull request merged**